src/cocaine-app/planner/__init__.py

```diff
@@ -30,11 +31,13 @@ class Planner(object):
     MOVE_CANDIDATES = 'move_candidates'
     RECOVER_DC = 'recover_dc'
     COUPLE_DEFRAG = 'couple_defrag'
+    TTL_CLEANUP = 'ttl_cleanup'
 
     RECOVERY_OP_CHUNK = 200
 
     COUPLE_DEFRAG_LOCK = 'planner/couple_defrag'
     RECOVER_DC_LOCK = 'planner/recover_dc'
+    TTL_CLEANUP_LOCK = 'planner/couple_ttl_cleanup'
```

Comment:
Done in https://github.com/yandex/mastermind/pull/311/commits/4732106cd00d7b48fb9a6db30b7cff4aef8db6df